system/Common.php

```diff
@@ -740,7 +740,7 @@ function old(string $key, $default=null)
 			$value = unserialize($value);
 		}
 
-		return $value;
+		return esc($value);
```

Comment:
Please add allow the user to specify the context for escaping. This could potentially be used in contexts other than straight HTML and the user should be able to correctly escape it. This requires adding a third argument to the method.